src/nvim/CMakeLists.txt

```diff
@@ -798,6 +798,10 @@ install(DIRECTORY ${BINARY_LIB_DIR}
   DESTINATION ${CMAKE_INSTALL_LIBDIR}/nvim/
   USE_SOURCE_PERMISSIONS)
 
+install(DIRECTORY ${LUAJIT_RUNTIME}
```

Comment:
This should only happen when building with the _bundled_ LuaJIT.

Reply:
Why? The main build is supposed to be independent of where luajit is, whether it's from the system or bundled.

Reply:
Because that directory is only available (and should be copied) if we are using the bundled LuaJIT. In all other situations, this will not work, nor be needed (see the failing PUC test).

Reply:
No, it's available when using the system luajit as well, at least on ubuntu.

Reply:
I've added an if, so this should work for PUC lua as well now.

Regarding the share files: I don't know how distributions typically handle `/usr/share` files, and whether we can expect them to ship these. If we only install these for bundled luajit then the behavior between neovim installed with bundled deps and neovim installed with system deps will change.

Should we bundle these files perhaps?

Reply:
No, the behavior should not change. If someone doesn't want to use the bundled LuaJIT, then it's their responsibility to install it and make sure _all_ files are visible to Neovim (e.g., by setting the default `package.path` to include the LuaJIT runtime directory, which is usually the case).

This is just about _us_ not following our own advice.